src/main/java/com/faforever/api/data/domain/Name.java

```diff
@@ -0,0 +1,47 @@
+package com.faforever.api.data.domain;
+
+import com.yahoo.elide.annotation.DeletePermission;
+import com.yahoo.elide.annotation.Include;
+import com.yahoo.elide.annotation.UpdatePermission;
+import lombok.Setter;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.FetchType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.ManyToOne;
+import javax.persistence.Table;
+import javax.validation.constraints.NotNull;
+import java.time.OffsetDateTime;
+
+@Entity
+@Table(name = "name_history")
+@Include(rootLevel = true, type = "name")
+@DeletePermission(expression = "Prefab.Role.None")
+@UpdatePermission(expression = "Prefab.Role.None")
+@Setter
+public class Name {
+  private OffsetDateTime changeTime;
+  private Player player;
+  private String name;
+
+  @Id
+  @Column(name = "change_time")
+  public OffsetDateTime getChangeTime() {
+    return changeTime;
+  }
```

Comment:
We need to refactor the DB to have an `int id` primary key.

Reply:
Seriously @IDragonfire it would haven taken you 5 more minutes to make the db and this would have been in production for a long time... https://github.com/FAForever/db/pull/137

Reply:
If I remeber correctly this is not the only place ...